tools/packaging/common/envoy_bootstrap.json

```diff
@@ -115,6 +115,22 @@
       {
         "tag_name": "authz_dry_run_result",
         "regex": "(\\.shadow_(allowed|denied))"
+      },
+      {
```

Comment:
Is part of this PR adding new default tags too?

Reply:
Yes. I'm trying to correct for some known issues as well. I can separate the PRs as needed.